depthai_ros_driver/src/common_stampers.cpp

```diff
@@ -0,0 +1,16 @@
+#include <depthai_ros_driver/common_stampers.hpp>
+
+#include <node_interface/ros1_node_interface.hpp>
+#include <nodelet/nodelet.h>
+
+namespace rr {
+template class ConverterAndStamper<depthai_ros_msgs::ImgDetectionsStamped,
+        ROS1Node<>>;  // ImgDetectionsStamper<ROS1Node<>>;
+template class ConverterAndStamper<depthai_ros_msgs::ImgDetectionsStamped,
+        nodelet::Nodelet>;  // ImgDetectionsStamper<nodelet::Nodelet>;
+
+template class ConverterAndStamper<depthai_ros_msgs::NNStamped,
+        ROS1Node<>>;  // NNStamper<ROS1Node<>>;
+template class ConverterAndStamper<depthai_ros_msgs::NNStamped,
+        nodelet::Nodelet>;  // NNStamper<nodelet::Nodelet>;
```

Comment:
we can remove the `//` now ?

Reply:
Sure. I kept the `//` to show that this long thing is. Apparently, you can't do `template class` with aliases.

Should I remove or keep the comments?